iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -56,30 +57,31 @@ public DeviceReconnectionSample(List<string> deviceConnectionStrings, TransportT
 
         public async Task RunSampleAsync(TimeSpan sampleRunningTime)
         {
-            using var cts = new CancellationTokenSource(sampleRunningTime);
+            s_cancellationTokenSource = new CancellationTokenSource(sampleRunningTime);
```

Comment:
creating a static variable so that we can access the cancellation token from within the status change callback

Reply:
What is the scope of things that will stop happening for this CTS? Perhaps we could get a better, more descriptive name?